### PR TITLE
Enable BF16 E2E tests

### PR DESCRIPTION
**Pull request by yiqian1**
These bf16 e2e tests have been run with -pv and -pv_with_gpu on lockhart6 and all passed.

### CODE REVIEW

File: mlir/test/e2e/Resnext101Config.toml [hunk 1]
```diff
@@ -13,7 +13,7 @@ values = ["-fil_layout=gkcyx -in_layout=ngchw -out_layout=ngkhw", "-fil_layout=g
 
 [[axis]]
 name = "data type"
-values = ["f32"]
+values = ["f32", "bf16"]
```
Comment (krzysz00) [inline mlir/test/e2e/Resnext101Config.toml]: I'm not sure we want to enable here since this is f32-only unlike the rest of them?

**Review by krzysz00 — COMMENTED** [review]
Do we want to get the PR tests as well or?

**Comment by yiqian1** [comment]
> Do we want to get the PR tests as well or?

Probably not, as bf16 is not critical at this moment? We're only testing f32 and f16 for PR now.

File: mlir/test/e2e/Resnext101Config.toml (hunk 1 above)
Comment (yiqian1) [reply mlir/test/e2e/Resnext101Config.toml]: > I'm not sure we want to enable here since this is f32-only unlike the rest of them?

Removed

**Review by krzysz00 — APPROVED** [review]

**Pull request merged**